engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -437,7 +435,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   }
 
   private def handleJobSuccessful(jobKey: JobKey, outputs: JobOutputs, data: WorkflowExecutionActorData) = {
-    workflowLogger.info(s"Job ${jobKey.tag} succeeded!")
+    workflowLogger.debug(s"Job ${jobKey.tag} succeeded!")
```

Comment:
Ooh, this might be controversial?

If you're not going to log this, you'll have a lot of `Starting calls:...` and no closing `Job succeeded!`